configuration-service/src/main/resources/configs/notification-service.yml

```diff
@@ -0,0 +1,4 @@
+eureka:
+  client:
+    service-url:
+      default-zone: http://localhost:8761/eureka
```

Comment:
Values that change depending on the environment (in this case: host, port) should be moved to be in the .env file.